src/GitVersionTask.Tests/GitVersionTask.Tests.csproj

```diff
@@ -55,36 +23,18 @@
     <PackageReference Include="Shouldly" Version="3.0.0"></PackageReference>
     <PackageReference Include="System.Reflection.Metadata" Version="1.4.2"></PackageReference>
     <PackageReference Include="System.Collections.Immutable" Version="1.3.1"></PackageReference>
-      <!--See https://github.com/Microsoft/msbuild/issues/3671 for reason for the ExcludeAssets against msbuild packagerefernces -->
-      <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.1.548" />
-      <PackageReference Include="Microsoft.Build.Framework" Version="15.1.548" />
-
-  </ItemGroup>
-  <ItemGroup>
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System" />
-    <Reference Include="System.ComponentModel.Composition" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Runtime.Serialization" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Xml" />
+    <!--See https://github.com/Microsoft/msbuild/issues/3671 for reason for the ExcludeAssets against msbuild packagerefernces -->
```

Comment:
If you follow the link above, I think there should be an excludeasset present here? Perhaps we should add it..

Reply:
you mean based on [this ](https://github.com/Microsoft/msbuild/issues/3671#issuecomment-418863446)comment?

Reply:
I will add that

Reply:
let's wait for the build to finish and if there is no other changes you think I should do we can merge it

Reply:
:-( looks like build failure

Reply:
I think it looks like for the tests project, they still need to be output, as it probably doesn't know how to resolve them, where as msbuild in the wild does..

Reply:
:-) probs best to add it back for the tests project then, and delete that confusing comment? I assume the comment must only be relevent for GitVersionTask.csproj!

Reply:
I'll do that

Reply:
removed the comment, reverted the change and squashed to have one commit